ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -728,6 +761,29 @@ void setChunkException(RouterException exception) {
     }
 
     /**
+     * Slice this chunk's data to only include the bytes within the operation's specified byte range.
+     * @param blobData the {@link BlobData} for this chunk.
+     * @return A {@link ByteBuffer} that only includes bytes within the operation's specified byte range.
+     */
+    ByteBuffer filterChunkToRange(BlobData blobData) {
```

Comment:
I can change it to `protected` if you think that communicates the intent better.  They have the same effect since the class is private.